app/assets/v2/js/pages/new_bounty.js

```diff
@@ -319,6 +339,19 @@ $(document).ready(function() {
         txid: null
       });
 
+      // Create Gitcoin tip transaction
+      const gitcoinTipAddress = '0x00de4b13153673bcae2616b67bf822500d325fc3';
```

Comment:
Wouldn't it be better to use an Environment variable by setting it in the settings.py? @mbeacom thoughts?

Reply:
I'm ready to do this if you want it :)

Reply:
yes please.

Reply:
I've got the environment variable setup in settings.py, what's the best way to get that from Python to the new_bounty.js file?

Reply:
You can send it in the context params from the view, and in new.html you can set it as an attribute of the global document variable in javascript.